@AH-64D Apache Longbow/Addons/fza_ah64_ku/config.cpp

```diff
@@ -0,0 +1,14 @@
+class CfgPatches
+{
+	class fza_ah64_ku
+	{
```

Comment:
needs updating to match control author list
required version needs updating
required addon needs updating